docs/retry-queueing.adoc

```diff
@@ -1,5 +1,5 @@
 = Retry Queuing
-:base_version: 0.0.40
+:base_version: 0.0.42
```

Comment:
please also bump other docs that has `:modules:` decl for `processor` to make a build passes.